### PR TITLE
Upgraded to bevy 0.14.0; All tests and examples passing

### PR DESCRIPTION
**Pull request by QueenOfSquiggles**
I imagine at first look this is a duplicate of #37 but with this PR I made sure tests were passing (Including doc tests) and the example (which acts as a decent integration test).

I was also tempted to add an `expect` call in the example to make panics more easily understood. Since the only reason the example *would* panic is if a steam client is not currently running as well. While it may be fairly obvious I think it could help to signal to newer users that what's being done isn't handling the potential for a panic.

For example in my use case I wish to distrubute across Steam and Itch IO. An Itch user shouldn't be expected to have a steam client running so I'd rather just handle it with some control statements.

Otherwise thanks a ton for making this crate! I'm really excited to integrate steamworks and this crate makes it super easy!

### CODE REVIEW

**Review by roylaurie — COMMENTED** [review]
In `example.rs`, I would either leave that as `.unwrap()` a make the error shorter for a panic (e.g., "Steam failed to initialize").

**Review by roylaurie — APPROVED** [review]
Working in my build

**Comment by QueenOfSquiggles** [comment]
Just awaiting review from a maintainer 🙃

**Comment by QueenOfSquiggles** [comment]
I'm closing this PR. I'm just going to use my own fork and keep that updated for my purposes. If anyone else wants to use that one, it should be relatively easy. I'll update the guidance at some point so bug me if I haven't gotten around to it yet

**Review by james7132 — COMMENTED** [review]
Sorry for the delay on this! I've been indisposed due to a whole host of reasons. This otherwise looks good to me.

**Review by james7132 — APPROVED** [review]

**Pull request merged**
